designs/virtualization/4.6/Snapshots-updates/snapshots-updates.md

```diff
@@ -0,0 +1,54 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Snapshots - updates
+
+## Adding a ‘Last restored’ column to the list view
+
+We'll add a ‘Last restored’ column to the snapshots list
+that will show which snapshot was restored most recently, so the latest snapshot would be at the top of the list and indicate the time when it was last restored from.
+
+![List view](img/Snapshots-list.png)
+
+## Restore VM from a snapshot
+
+After the user clicked to ‘Restore’ VM from a snapshot we will show in the status column that the VM is currently being restored from this snapshot.
+Restore can be done either from the kebab menu or from the secondary button at each row.
+
+![Restoring](img/Restore-from-kebab.png)
```

Comment:
can you confirm that it was implemented this way? I thought we revised the status of the snapshot to just be ready or show an error but not the progress of the restore (We'd leave that to the status of the VM). I might be wrong though :)

Reply:
Just confirmed with @glekner and we do show restore, usually it's not seen because this action is instant.